modules/mono/editor/bindings_generator.h

```diff
@@ -179,6 +187,14 @@ class BindingsGenerator {
 		 */
 		bool is_internal = false;
```

Comment:
We might want to update the comment on this, it speaks of setters and getters but are not internal any longer

Reply:
I think this still applies, it refers to property getters and setters when the method starts with an underscore and are not virtual. For example: `AnimationLibrary::_set_data`.